docs/system-admin-guide/design/README.md

```diff
@@ -60,16 +65,16 @@ Click the *Upload* button to confirm and upload your custom touch icon.
 
 When you bookmark your OpenProject environment’s URL, you will see that the uploaded icon is used as a custom touch icon.
 
-## Advanced settings
+## Interface colors
 
 Aside from uploading logos and icons, you can also customize the colors used within your OpenProject environment.
 
-To do this change the color values (entered as color hex code) in the *Advanced settings* section. In order to find the right hex code for a color, you can use a website, such as [color-hex.com](https://www.color-hex.com/).
- You can see the selected color in the preview area next to the color hex code. Therefore, it is possible to see the selected color before saving the changes.
+To do this change the color values (entered as color hex code) under *Interface* tab. In order to find the right hex code for a color, you can use a website, such as [color-hex.com](https://www.color-hex.com/).
```

Comment:
Suggestion (this website makes it easier to just pick a colour from a basic picker):

> To do this, enter the hex value for any color you would like to change. You can use a website like [htmlcolorcodes.com](https://htmlcolorcodes.com/color-picker/) to help you find the perfect color.